packages/pn-pa-webapp/src/redux/apiKeys/reducers.ts

```diff
@@ -1,10 +1,21 @@
-import { createSlice } from '@reduxjs/toolkit';
-import { ApiKey } from '../../models/ApiKeys';
+import { PayloadAction, createSlice } from '@reduxjs/toolkit';
+import { ApiKeys } from '../../models/ApiKeys';
+import { UserGroup } from '../../models/user';
 import { getApiKeys } from './actions';
 
 const initialState = {
   loading: false,
-  apiKeys: [] as Array<ApiKey>,
+  apiKeys: {
+    items: [],
+    total: 0,
+  } as ApiKeys<UserGroup>,
+  pagination: {
```

Comment:
in pagination we can store the size (= limit), the page and an array of object like that

`{lastKey: ..., lastUpdate: ...}`

Reply:
changes made into 5eb41f24bacd7862f6c9ac99ae9f9e9396e71a40